packages/http/lib/http-decorators.tsp

```diff
@@ -51,7 +51,7 @@ model QueryOptions {
   /**
    * Determines the format of the array if type array is used.
    */
-  format?: "multi" | "csv" | "ssv" | "tsv" | "pipes";
+  format?: "multi" | "csv" | "ssv" | "tsv" | "simple" | "form" | "pipes";
```

Comment:
I think we need some docs to explain all those formats. Specially the equivalence between simple-csv and multi-form

Reply:
Added parameter docstring for this.